hs_tracking/middleware.py

```diff
@@ -45,7 +47,7 @@ def get_landing_from_url(path):
         return False
 
 
-class Tracking(object):
+class Tracking(MiddlewareMixin):
```

Comment:
This mixin is to provide compatibility of MIDDLEWARE setting (1.10+) with MIDDLEWARE_CLASSES setting(<1.10)